pysat/instruments/cosmic2013_gps.py

```diff
@@ -180,10 +198,10 @@ def load(cosmicFiles, tag=None, sat_id=None, altitude_bin=None):
 # becuase I was playing around with multiprocessor loading
 # yielded about 20% improvement in execution time
 def load_files(files, tag=None, sat_id=None, altitude_bin=None):
-    '''Loads a list of COSMIC data files, supplied by user.
+    """Loads a list of COSMIC data files, supplied by user.
```

Comment:
Needs improved docstring

Reply:
fixed here and in cosmic_gps.  For consistency, also renamed output variable as data